velox/functions/prestosql/tests/InPredicateTest.cpp

```diff
@@ -328,3 +328,156 @@ TEST_F(InPredicateTest, reusableResult) {
   auto expected = makeFlatVector<bool>({false, true, true, false});
   assertEqualVectors(expected, actual);
 }
+
+class InPredicateDecimalTest : public FunctionBaseTest {
```

Comment:
The values in the IN-list are hardcoded and the input vector is constructed inside the function so they cannot be re-used currently. A common test utility function with a template variable can be used as attempted before for short and long decimals, but it could affect the readability.
Please let me know if I should use a common function.